BlinkID/android/src/main/java/com/microblink/flutter/MicroblinkCreationParams.kt

```diff
@@ -0,0 +1,40 @@
+package com.microblink.flutter
```

Comment:
I'm not sure about bringing Kotlin to Java-only project.

Ideally, I would like to merge our changes into upstream, so that we don't diverge completely in the end.

Reply:
You are right. Thought I could get away not having to use Java 😆. I've moved switched to Java, although not sure if the implementation adhere to Android guidelines.